test/pretty.test.js

```diff
@@ -367,6 +367,24 @@ test('works without time', function (t) {
   instance.info('hello world')
 })
 
+test('convert timestamp to local-time with tz', function (t) {
+  t.plan(5)
+  var prettier = pretty()
+  prettier.pipe(split(function (line) {
+    let now = line.slice(line.indexOf('>') + 1, line.indexOf('<'))
+    t.ok(line.match(/.*hello world$/), 'end of line matches')
+    t.ok(line.match(/(?!^)INFO.*/), 'includes level')
+    t.ok(line.indexOf('' + process.pid) > 0, 'includes pid')
+    t.ok(line.indexOf('' + hostname) > 0, 'includes hostname')
+    t.ok(Date.parse(line.substr(1, 29)) === parseInt(now), 'local date timezone')
+    return line
+  }))
+
+  var instance = pino(prettier)
+
+  instance.info('>' + Date.now() + '<' + ' hello world')
```

Comment:
This Date.now() may not be doing what you think. Can you remove it?

Reply:
````js
instance.info('>' + Date.now() + '<' + ' hello world')
var now = line.slice(line.indexOf('>') + 1, line.indexOf('<'))
t.ok(Date.parse(line.substr(1, 29)) === parseInt(now), 'local date timezone')
````
According to MDN, `Date.now()` will return the number of milliseconds elapsed since January 1, 1970 00:00:00 UTC. right?
this is what I try to do
```
[2018-02-10T17:32:02.719+08:00] INFO (8484 on jimmy): >1518255122719< hello world
Date.parse('2018-02-10T17:32:02.719+08:00') === 1518255122719
           ^^^ local datetime with tz ^^^
```

Reply:
I don't think this is a good way to test this. You are assuming that the time picked from `instance.info()` is the same of the log message. However, this is not guaranteed. 
A better approach might be to mock `Date.now()`.